src/framework/mlt_repository.h

```diff
@@ -26,6 +26,12 @@
 #include "mlt_types.h"
 #include "mlt_profile.h"
 
+#ifdef _WIN32
+#  define LIBSUF ".dll"
+#else
+#  define LIBSUF ".so"
+#endif
+
```

Comment:
Did you intend to leave `LIBSUF` macro defined here? I am not so sure that is relevant. frei0r is the rare exception that a module needs to use dlopen, and like I said for macOS it might not even be correct for another module's need.